action.yml

```diff
@@ -5,13 +5,29 @@ branding:
   icon: check-circle
   color: yellow
 
+inputs:
+  is_windows:
```

Comment:
Can’t we detect this rather than requiring an input?

Reply:
Indeed we can! Fixed.